test/runtests.jl

```diff
@@ -16,16 +16,19 @@ using OpticSim
 # Geometry Module
 using OpticSim.Geometry
 # curve/surface imports
-using OpticSim: findspan, makemesh, knotstoinsert, coefficients, inside, quadraticroots, tobeziersegments, evalcsg, makiemesh
+using OpticSim: makemesh, inside
+using OpticSim.Core: findspan, knotstoinsert, coefficients, quadraticroots, tobeziersegments, evalcsg, makiemesh
 # interval imports
-using OpticSim: α, halfspaceintersection, positivehalfspace, lower, upper, EmptyInterval, rayorigininterval, intervalcomplement, intervalintersection, intervalunion, RayOrigin, Infinity, Intersection
+using OpticSim: EmptyInterval, RayOrigin, Infinity, Intersection
```

Comment:
The items used from OpticSim (EmptyInterval, etc.) seem to be things that would be defined in OpticSimCore from which you import a bunch of other stuff related to rays and ray tracing. Was there a reason you broke them apart like this?